app/models/web_hook_event_type.rb

```diff
@@ -16,6 +16,7 @@ class WebHookEventType < ActiveRecord::Base
   LIKE = 15
   USER_PROMOTED = 16
   TOPIC_VOTING = 17
+  CHAT_MESSAGE = 18
```

Comment:
I've chosen `CHAT_MESSAGE` instead just `CHAT` in case in the future we decide to add webhooks to track CHAT_CHANNEL events